codegen/kuberesource/crd.go

```diff
@@ -142,6 +164,16 @@ func CustomResourceDefinition(
 			},
 		},
 	}
+	if !options.WithoutSpecHash {
+		// hashstructure of the crd spec:
+		specHash, err := hashstructure.Hash(crd.Spec, nil)
+		if err == nil {
+			crd.Annotations = map[string]string{
+				model.CRDSpecHashKey: fmt.Sprintf("%x", specHash),
+			}
+		}
```

Comment:
log an error if hashing fails

Reply:
good point - I think i will actually panic, as this should never happen.